airflow/www/static/js/dag/StatusBox.tsx

```diff
@@ -36,15 +36,32 @@ import InstanceTooltip from './InstanceTooltip';
 export const boxSize = 10;
 export const boxSizePx = `${boxSize}px`;
 
-interface SimpleStatusProps extends BoxProps {
+interface StatusWithNotesProps extends BoxProps {
   state: TaskState;
+  containsNotes?: boolean;
 }
+export const StatusWithNotes = ({ state, containsNotes, ...rest }: StatusWithNotesProps) => {
+  const theColor = state && stateColors[state] ? stateColors[state] : 'white';
```

Comment:
Can we get a more descriptive name please?

Reply:
Changed `theColor` to `backgroundColor`